packages/core/src/components/datetime/datetime.scss

```diff
@@ -0,0 +1,102 @@
+.atom-datetime {
+  &::part(calendar-day) {
+    position: relative;
+
+    &:hover {
+      &:after {
+        background-color: rgba(var(--color-neutral-black-rgb), 0.07);
+        border-radius: 50%;
+        content: '';
+        height: 100%;
+        left: 0;
+        position: absolute;
+        top: 0;
+        width: 100%;
+      }
+    }
+  }
+
+  &::part(calendar-day active) {
+    background-color: red;
+  }
+}
+
+.atom-item {
+  position: relative;
+}
+
+.atom-label {
+  color: var(--color-neutral-regular);
+  font: var(--text-caption);
+  left: 0;
+  letter-spacing: var(--text-caption-letter);
+  max-width: calc(100% / 0.75);
+  overflow: hidden;
+  padding-left: var(--spacing-base);
+  position: absolute;
+  text-overflow: ellipsis;
+  top: 7px;
+  white-space: nowrap;
+  z-index: var(--zindex-5);
+}
+
+.atom-icon {
+  color: var(--color-neutral-regular);
+  pointer-events: none;
+  position: absolute;
+  right: var(--spacing-base);
+  top: 50%;
+  transform: translateY(-50%);
+  z-index: var(--zindex-5);
+}
+
+.atom-button {
+  border-bottom: 1px solid
+    var(--ion-color-step-500, var(--ion-background-color-step-500, gray));
+  width: 100%;
+
+  .atom-item:hover & {
```

Comment:
is there an extra `&`? :thinking:

Reply:
Nops! The propose here is catch the `atom-button` when it is inside `.atom-item:hover`, this is the generated CSS

```css
.atom-item:hover .atom-button::part(native) {
  background: var(--ion-color-step-100, var(--ion-background-color-step-100, #e6e6e6));
}
```

Reply:
daammn, I didn't know that. Thanks!